### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by qhashi — APPROVED** [review]
Looks a okay

File: RatersOfTheLostBusiness/RatersOfTheLostBusiness/Models/Services/BusinessService.cs [hunk 1]
```diff
@@ -57,8 +57,13 @@ public async Task<BusinessSmsDto> GetBusinessByName(string name)
...
+                    //ADD MAYBE: CUMULATIVE RATING
+                    Review = b.BusinessReviews
+                    .Select(br => new BusinessReviewDto
+                    {
+                        Rating = br.Rating,
+                        Review = br.Review
+                    }).ToString()
                 }).FirstOrDefaultAsync(b => b.Name == name);
         }
         public async Task<List<BusinessDto>> GetBusinesses() // async
```
Comment (BeniBarra) [inline RatersOfTheLostBusiness/RatersOfTheLostBusiness/Models/Services/BusinessService.cs:69]: Remember to remove old/commented code, as seen between lines 81-88.

File: RatersOfTheLostBusiness/TestProject1/MockTest.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using RatersOfTheLostBusiness.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RatersOfTheLostBusiness.Models;
+
```
Comment (BeniBarra) [inline RatersOfTheLostBusiness/TestProject1/MockTest.cs:8]: Could benefit from delete, and update tests. Great job on the post test!

File: RatersOfTheLostBusiness/RatersOfTheLostBusiness/Data/BusinessDbContext.cs [hunk 3]
```diff
@@ -26,19 +26,30 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
```
Comment (BeniBarra) [inline RatersOfTheLostBusiness/RatersOfTheLostBusiness/Data/BusinessDbContext.cs:26]: Remember to write DBsets with PascalCase

File: RatersOfTheLostBusiness/RatersOfTheLostBusiness/Data/BusinessDbContext.cs [hunk 4]
```diff
@@ -26,19 +26,30 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
                 new Business
                 {
-                    Id = 1,
-                    Name = "Twilio",
-                    Address = "375 Beale Street Suite 300",
-                    City = "San Franciso",
-                    State = "CA",
-                    PhoneNumber = "844-814-4627",
-                    Type = "Software Service"
+                    Id = 1, Name = "Twilio", Address = "375 Beale Street Suite 300", City = "San Franciso", State = "CA", PhoneNumber = "844-814-4627", Type = "Software Service"
```
Comment (BeniBarra) [inline RatersOfTheLostBusiness/RatersOfTheLostBusiness/Data/BusinessDbContext.cs:34]: new Object constructor conventions were inconsistent. Some were written inline, and others in rows.

File: RatersOfTheLostBusiness/RatersOfTheLostBusiness/Controllers/BusinessSmsController.cs [hunk 5]
```diff
@@ -83,7 +83,7 @@ public async Task<TwiMLResult> ReceiveSms()
             if (userInput == business.Name)
             {
                 //return the corresponding name, address and rating
-                responseToUser.Message($"{business.Name} is located at {business.Address} and has a rating of {business.Rating}");
+                responseToUser.Message($"{business.Name} is located at {business.Address} the last review said: {business.ReviewSms.Review} ");
             }
 
             return TwiML(responseToUser);
```
Comment (BeniBarra) [inline RatersOfTheLostBusiness/RatersOfTheLostBusiness/Controllers/BusinessSmsController.cs:89]: A bit more clean up of old/commented code between lines 90-114

**Review by BeniBarra — COMMENTED** [review]
Generally all the code looked fine. The few things found were old/commented code, as well as some inconsistent conventions for object constructors, while not the end of the world it would be nice to see them uniform. Over all great job!

**Pull request merged**